templates/kernel/variants.html

```diff
@@ -148,8 +148,8 @@ <h2 id="current-variant-kernels">Current variant kernels</h2>
       <h2 id="additional-kernel-variants">Additional kernel variants</h2>
       <p>In addition to those kernel variants, Canonical offers FIPS certified kernels:</p>
       <ul>
-        <li>FIPS &mdash; Certified GA kernel for Ubuntu 16.04 and 18.04</li>
-        <li>FIPS &mdash; compliant GA kernel with critical security and patch updates for Ubuntu 16.04 and 18.04</li>
+        <li>FIPS &mdash; Certified GA kernel for Ubuntu 16.04, 18.04, and 20.04</li>
+        <li>FIPS &mdash; compliant GA kernel with critical security and patch updates for Ubuntu 16.04, 18.04, and 20.04</li>
```

Comment:
I *think* we shouldn't be using the serial comma here. See https://webteam.canonical.com/practices/copy-reviews:

> Do enumerations use the serial comma? (They shouldn’t)
> 
>     Correct: We design, build and run your private cloud.
>     Incorrect: We design, build, and run your private cloud.

I'll update it.

```suggestion
        <li>FIPS &mdash; Certified GA kernel for Ubuntu 16.04, 18.04 and 20.04</li>
        <li>FIPS &mdash; compliant GA kernel with critical security and patch updates for Ubuntu 16.04, 18.04 and 20.04</li>
```